Misc/NEWS.d/next/C API/2020-09-11-02-50-41.bpo-14916.QN1Y03.rst

```diff
@@ -0,0 +1,2 @@
+fixed bug in "The Very High Level Layer" of c-api that prevented
+PyRun_InteractiveOne*() parsing from the provided FD.
```

Comment:
```suggestion
PyRun_InteractiveOne\*() parsing from the provided FD.
```

GitHub already tries to interpret a star as a directive lasting until the end of a line. To be safe with pydoc generator I think it's better to escape it.